WalletWasabi.Fluent/Views/SearchBar/SearchBar.axaml.cs

```diff
@@ -14,4 +16,12 @@ private void InitializeComponent()
 	{
 		AvaloniaXamlLoader.Load(this);
 	}
+
+	public void Unfocus()
+	{
+		if (Application.Current?.ApplicationLifetime is IClassicDesktopStyleApplicationLifetime { MainWindow: { } mainWindow })
+		{
+			mainWindow.Focus();
+		}
+	}
```

Comment:
IMO we should have a flag like `SearchBar.IsActive` and we should just modify that one value. And when it is false, the flyout should close and SearchBar should lose focus.

And everything should control that one flag. It is getting complex to handle every edge case.
Anyway, this is out of the scope of this PR.